compose/grafana/dashboards/openwhisk_events.json

```diff
@@ -119,7 +119,7 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "sum(increase(openwhisk_action_activations_total{namespace=~\"$namespace\",action=~\"$action\"}[$interval]))",
+          "expr": "sum(increase(openwhisk_action_activations_total{stack=\"$stack\",namespace=~\"$namespace\",action=~\"$action\"}[$interval]))",
```

Comment:
Current code does not emit `stack` attribute directly. In our internal setup those are added by the agents